src/factories/workflow-specific/Immutable_PIM_Factory_v1.sol

```diff
@@ -0,0 +1,184 @@
+                memory bcProperties,
+            address collateralToken
+        ) = abi.decode(
+            fundingManagerConfigData,
+            (
+                address,
+                IFM_BC_Bancor_Redeeming_VirtualSupply_v1.BondingCurveProperties,
+                address
+            )
+        );
+        fundingManagerConfigData =
+            abi.encode(address(issuanceToken), bcProperties, collateralToken);
+        fundingManagerConfig.configData = fundingManagerConfigData;
+        orchestrator = IOrchestratorFactory_v1(orchestratorFactory)
+            .createOrchestrator(
+            workflowConfig,
+            fundingManagerConfig,
+            authorizerConfig,
+            paymentProcessorConfig,
+            moduleConfigs
+        );
+        // get bonding curve / funding manager
+        address fundingManager = address(orchestrator.fundingManager());
+        // enable bonding curve to mint issuance token and disable minting from factory
+        issuanceToken.setMinter(fundingManager, true);
+        issuanceToken.setMinter(address(this), false);
+
+        // if initial purchase amount is set (> 0) execute first purchase from curve
+        // recipient: initiator
+        if (initialPurchaseAmount > 0) {
+            IERC20(collateralToken).transferFrom(
+                _msgSender(), address(this), initialPurchaseAmount
+            );
+            IERC20(collateralToken).approve(
+                fundingManager, initialPurchaseAmount
+            );
+            IBondingCurveBase_v1(fundingManager).buyFor(
+                initiator, initialPurchaseAmount, 1
+            );
+        }
+
+        // set fee recipient (initiator)
+        _pimFeeRecipients[fundingManager] = initiator;
+
+        // renounce token ownership
+        issuanceToken.renounceOwnership();
+
+        emit IImmutable_PIM_Factory_v1.PIMWorkflowCreated(
+            address(orchestrator), address(issuanceToken), _msgSender()
+        );
+    }
+
+    //--------------------------------------------------------------------------
+    // Permissioned Functions
+
+    /// @inheritdoc IImmutable_PIM_Factory_v1
+    function withdrawPimFee(address fundingManager, address to)
+        external
+        onlyPimFeeRecipient(fundingManager)
+    {
+        // get accumulated fee amount from bonding curve
+        uint amount =
+            IBondingCurveBase_v1(fundingManager).projectCollateralFeeCollected();
+        // withdraw fee from bonding curve and send to `to`
+        IBondingCurveBase_v1(fundingManager).withdrawProjectCollateralFee(
+            to, amount
+        );
+        emit IImmutable_PIM_Factory_v1.PimFeeClaimed(
+            fundingManager, _msgSender(), to, amount
+        );
+    }
+
+    /// @inheritdoc IImmutable_PIM_Factory_v1
+    function transferPimFeeEligibility(address fundingManager, address to)
+        external
+        onlyPimFeeRecipient(fundingManager)
+    {
+        _pimFeeRecipients[fundingManager] = to;
```

Comment:
Saw the thumbs-up here but nothing has changed, right? Which is fine, just wanna make sure that we don't forget about it, in case you intended to add something here. If you think this should be left like it is, that's also fine.

Reply:
Sorry, I overlooked this. Added a check now.